Snakemake/rules/prepare_files_for_tsinfer.smk

```diff
@@ -37,16 +37,52 @@ if noVCFs == 1:
             bcftools index {output.vcf}
```

Comment:
Here you are using 'chromosome' as a wildcard defined by a list of numbers. How will it handle sex chromosomes? 
Will we have to recode them (say X = Chr30)? We could define the whole 'Chr{n}' as a wildcard.

Reply:
Hm, maybe I can make an argument "chromosomes" in the .yaml file? Or maybe have another argument "numeric chromosomes" and "non-numeric chromosomes"?

Reply:
Yes. We can talk about it on the next meeting. But it can be defined with regular expression to say, for example, 'Chr' + anything number or letter until '_' (and we define this as a standard name).